ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/TransactionDecoder.java

```diff
@@ -60,24 +60,30 @@ interface Decoder {
       Suppliers.memoize(SignatureAlgorithmFactory::getInstance);
 
   public static Transaction decodeForWire(final RLPInput rlpInput) {
+    // Call the original decodeForWire method with the default options.
+    return decodeForWire(rlpInput, DecodeType.DEFAULT);
+  }
+
+  public static Transaction decodeForWire(final RLPInput rlpInput, final DecodeType decodeType) {
```

Comment:
Does this method name make sense anymore, since I can decodeForWire but pass in DecodeType.DEFAULT which is used when decoding Blob txs from a block which only contain versionedHashes, and is NOT what we have meant before with "forWire" which used to indicate to be RLP's out via gossip.